tests/system_info.c

```diff
@@ -54,6 +57,35 @@ static void print_platform_info(void) {
     stmt_c;
 #endif
 
+/* Display all active CPU extensions: */
+static void print_cpu_extensions(void) {
+#if defined(OQS_USE_CPU_EXTENSIONS) && defined(OQS_PORTABLE_BUILD)
+	/* Make CPU features struct iterable */
+	typedef union ext_u {
+		OQS_CPU_EXTENSIONS ext_x;
+		unsigned int ext_a[sizeof(OQS_CPU_EXTENSIONS) / sizeof(unsigned int)];
+	} OQS_CPU_EXTENSIONS_UNION;
+
+	OQS_CPU_EXTENSIONS_UNION ext_u;
+	ext_u.ext_x = OQS_get_available_CPU_extensions();
+	printf("CPU exts active:  ");
+	unsigned int it = sizeof(ext_u.ext_a) / sizeof(ext_u.ext_a[0]);
+	for (unsigned int i = 0; i < it; i++) {
+		if (ext_u.ext_a[i]) {
+			printf("%s", OQS_get_cpu_extension_name(i));
+		} else {
+			printf("n/a");
```

Comment:
I don't know if it's necessary to print "n/a" for ones that are disabled.

Reply:
Agree -- looked weird. Gone.